Dockerfile

```
@@ -0,0 +1,10 @@
+FROM golang:alpine as builder
+RUN mkdir /build
+ADD . /build/
+WORKDIR /build
```

```suggestion
WORKDIR /build
ADD . .
```

Slight change for brevity - Docker will implicitly create the WORKDIR if it doesn't already exist, so this saves a step/layer in the process.